packages/extension-chakra-storefront/src/components/field/index.jsx

```diff
@@ -64,95 +63,109 @@ const Field = ({
                     typeof inputProps === 'function' ? inputProps({value, onChange}) : inputProps
 
                 return (
-                    <FormControl id={name} isInvalid={error}>
-                        {!['checkbox', 'radio', 'hidden'].includes(type) &&
-                            (formLabel || <FormLabel>{label}</FormLabel>)}
+                    <Field.Root name={name} isInvalid={!!error} isRequired={!!rules?.required}>
```

Comment:
There's no longer `isInvalid` in chakra v3. It's now `invalid`.

Reply:
Got it.. also updated `isRequired` to `required`